instance/serializers/instance.py

```diff
@@ -71,23 +71,23 @@ class Meta:
     def to_representation(self, obj):
         output = super().to_representation(obj)
         # Convert the state values from objects to strings:
+        if output['server_status'] is None:
+            output['server_status'] = 'empty'  # 'empty' for backwards compatibility
```

Comment:
If we're changing the key name, we could also change the value here - unless `empty` is more convenient than `null` for the client-side code.